dokan/dokan.c

```diff
@@ -570,44 +566,20 @@ DispatchCommon(PEVENT_CONTEXT EventContext, ULONG SizeOfEventInfo,
   return eventInfo;
 }
 
-PDOKAN_OPEN_INFO
-GetDokanOpenInfo(PEVENT_CONTEXT EventContext, PDOKAN_INSTANCE DokanInstance) {
-  PDOKAN_OPEN_INFO openInfo;
-  EnterCriticalSection(&DokanInstance->CriticalSection);
-
-  openInfo = (PDOKAN_OPEN_INFO)(UINT_PTR)EventContext->Context;
-  if (openInfo != NULL) {
-    openInfo->OpenCount++;
-    openInfo->EventContext = EventContext;
```

Comment:
@Liryna, my patch is also removing this line in particular which I'm not sure is a good idea. Do you know why the per-thread EventContext pointer is stored in DOKAN_OPEN_INFO pointer while it is shared by multiple threads? This looks strange to me but the code might have been relying on this. I will see if this EventContext pointer can be moved out of the DOKAN_OPEN_INFO struct.